pkg/restclient/common.go

```diff
@@ -102,7 +96,9 @@ type Client struct {
 // to send request.
 func NewClientWithOpts(opts ...ClientFunc) (*Client, error) {
 	client := &Client{
-		httpClient: http.DefaultClient,
+		httpClient: &http.Client{
+			Transport: http.DefaultTransport,
```

Comment:
I think as a default configure you also need to set these values,
```
DisableCompression: true,
DisableKeepAlives:  true
```

Reply:
@aravindavk @prashanthpai  why we are disabling keep alives?  it will prevent user from re-using of TCP connections
between different HTTP requests. But we can reuse the TCP connection.